docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
It would be nice to have a management command to build and optionally open the docs, e.g. `python manage.py docs --browse`. 

That can be done in a separate PR, though.